UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -281,6 +281,32 @@ public void CmdDropAllItems(uint itemSlotID, Vector3 Target)
 		}
 	}
 
+	/// <summary>
+	/// Transfers x amount of items from one hand to another. For stackable items only
+	/// </summary>
+	[Command]
+	public void CmdSplitStack(uint fromSlotID, uint toSlotID, NamedSlot fromSlot, NamedSlot toSlot, int amountToTransfer)
```

Comment:
Pretty sure you wont need to send this, only which the "to hand" and the amount since the from will always be the active hand?

Reply:
Made this for clarity. Also, Ill still need to get hand slot containing stack even if I know which hand is active, doesn't really matter if Ill do this check in network actions or in menu itself

Reply:
nvm, gonna fix it

Reply:
Fixed in new commit